### DISCUSSION TITLE
Tutorials would be a good idea.

### DISCUSSION THREAD

**Discussion by SeleDreams** [friction2d/friction#192] [Ideas]
I think that to bring more users to the software, it might be a good idea to make tutorials how to achieve common types of animations to get users started. Right now the software might seem quite daunting for beginners.

**Comment by rodlie** [comment]
Tutorials would be great, but someone has to make them... ;) I don't have time to do that.

Friction is a tool by/for people that have experience with animation software and need little help getting started.

There are plans to make some example projects etc for the next version (v1).

**Comment by Chepycou** [comment]
Video tutorials would definitely be a plus, since it's easier to follow + it brins attention to the project

**Comment by rodlie** [comment]
Tutorials will at some point happen, but it's mostly up to the community to provide them.

The next version will at least include some example projects and maybe some documentation.

Since Friction is a fork you can find videos for the original project (enve), some things have changed, but the core concepts are still the same.

* https://www.youtube.com/playlist?list=PLQFW7x3qFlrBAqVFHCS8QGf2ooegv4riN
* https://www.youtube.com/playlist?list=PLiIhhu566LD9xzcRXgRqMkr8sIqaKO1HF
* https://www.youtube.com/results?search_query=enve+2d

**Comment by magdesign** [comment]
used this software for the first time (to create a postmarketOS bootsplash) and i love it!
its very intuitive, i am wondering what kind of tutorials are you guys looking for concretely ?

(so i can think about and create one to say thanks to rodlie...)

**Reply by rodlie** [reply]
Hi,

We are having a discussion related to documentation and tutorials [here](https://github.com/friction2d/friction/issues/208).

I think beginner tutorials would be a good start. How does the app work? What can you make with it? and then go from there.

I also recommend not doing anything until v1 beta drops (very soon), many changes since 0.9.

**Comment by rodlie** [comment]
Closing this *(to avoid fragmented discussion)*, further discussion around documentation, tutorials and demo files happens [here](https://github.com/friction2d/friction/issues/208).
